src/main/java/uk/gov/indexer/IndexerTask.java

```diff
@@ -28,9 +28,7 @@ public IndexerTask(Optional<CloudwatchRecordsProcessedUpdater> cloudwatchUpdater
     @Override
     public void run() {
         try {
-            LOGGER.info("Starting update for: " + register);
             update();
-            LOGGER.info("Finished for register: " + register);
         } catch (Throwable e) {
```

Comment:
Om and I discussed this - my view is that removing such a line means that you'd no longer know how long an update takes.

```
2016-03-08T14:25:45.211+0000 [pool-1-thread-9] INFO uk.gov.indexer.IndexerTask - Starting update for: uk
2016-03-08T14:25:45.239+0000 [pool-1-thread-9] INFO uk.gov.indexer.IndexerTask - Finished for register: uk
```

If we want to remove these lines, I would suggest we want another mechanism to report how long each update takes, so you can proactively identify slowdowns in the update process before they occur.